### ISSUE TITLE
Compatibility with Jetson Nano 4GB

### ISSUE THREAD

**Issue by ghost** [gante/mmWave-localization-learning#32]
Trying to run an experiment on Jetson Nano (4GB version) with all the requirements installed, but I keep receiving the error ''MemoryError'' on terminal. Is that a hardware problem of the Jetson Nano (inefficient memory)?
Note: I also increased the size of the swap memory (I use a 32GB SD card) , but it didn't solve the problem.

**Comment by gante** [comment]
Hey @konarva 👋 by "running the experiment", I'm assuming you mean doing inference using a model trained somewhere else :)

I ran the inference experiment long ago, so my ability to help will be limited. The model itself should be much smaller than 4GB, so I don't think it is the issue. The python environment, however, might have changed significantly 🤔 I'd suggest trying to ensure you use the same packages as in [`requirements.txt` from this specific commit](https://github.com/gante/mmWave-localization-learning/tree/6220366084fba85a0bc3dc8b0aff26ce875c0713), which corresponds to the codebase at the time of the experiment :)

**Comment by gante** [comment]
Yeah, the first two steps must be executed in a separate machine to train the model :) After you have a trained model, copy it to the Jetson machine and run the `test_model.py` step
